commands/timeout.js

```diff
@@ -24,58 +36,52 @@ module.exports = {
       return;
     }
 
-    for (var [id, user] of message.mentions.users) {
```

Comment:
Nobody was ever aware it could handle more than one at once, and it barely gets used. It just makes the code easier to manage if we get rid of it tbh.